### ISSUE TITLE
JT65

### ISSUE THREAD

**Issue by Sergeapm** [jks-prv/Beagle_SDR_GPS#110]
It possibly  to do?
Decoder JT65 into KiwiSDR?
Thanks

**Comment by jks-prv** [comment]
It's probably as much work as the WSPR extension. Which means a lot. Not sure about the processing requirements which might be the only difficulty.

**Comment by Sergeapm** [comment]
OK   Understand  ,,,  but I am sure  decoders  like HF FAX  WSPR   CW  JT65  and so on  will  be only  to do more popular  your  nice project  KiwiSDR

**Comment by jks-prv** [comment]
Agree, but there is only so much I can do. Already working 12/7. The full bug/wish list is here: kiwisdr.com/bugs

**Comment by Sergeapm** [comment]
of course  and many thanks for you hard work  all  very good  73!

**Comment by YO3IBZ** [comment]
@jks-prv if they ( https://github.com/jketterl/openwebrx ) can decode JT65, FT8, etc. why we can't?

Thank's,
Aurel.

**Comment by jks-prv** [comment]
OpenWebRX has little to do with KiwiSDR these days. They are written in different languages, have different server back-ends etc.

I have not looked to see how they did it. But remember that the server-side of OpenWebRX runs on a fully-featured PC where there is a lot more processing power (server-side = place where you plug your RTL-SDR into). They could be running a WSJT-X to do the FT8 decoding that is only modestly modified to interface to the OpenWebRX user interface.

The Kiwi doesn't have this luxury. The Beagle has limited processing power and real-time constraints. Despite this I was able to get the core parts of the WSPR decoder working after much modification to the code from WSJT. It took a long time to do this.

And I don't have the time right now for another long project. DRM took me many, many months. And I'm not personally interested in JT65 and FT8 that much.

**Comment by YO3IBZ** [comment]
@jks-prv yes BB has a hardware limitation, i understand and i appreciate your hard work!

Thank you for your reply!
Aurel.

**Comment by jks-prv** [comment]
Partially fixed, as we have FT8/FT4 decoding now.
